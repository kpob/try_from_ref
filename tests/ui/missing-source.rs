#![no_main]

#[derive(derive_try_from::TryFromRef)]
#[err(syn::Error)]
struct MyStruct {
    field: u32
}