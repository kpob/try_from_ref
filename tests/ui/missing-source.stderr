error: Missing source attribute

         = help: Add #[source(Type)] attribute

 --> tests/ui/missing-source.rs:5:8
  |
5 | struct MyStruct {
  |        ^^^^^^^^
